tap_harvest/__init__.py

```diff
@@ -137,6 +137,22 @@ def request(url, params=None):
     return resp.json()
 
 
+# Any date-times values can either be a string or a null.
+# If null, parsing the date results in an error.
+# Instead, removing the attribute before parsing ignores this error.
+def remove_empty_date_times(item, schema):
+    fields = []
+
+    for key in schema['properties']:
+        d = schema['properties'][key]
+        if d.get('format') == 'date-time':
+            fields.append(key)
+
+    for field in fields:
+        if item.get(field) is None:
+            del item[field]
```

Comment:
Yes, good point. It does appear that there is an issue in the transform code of https://github.com/singer-io/singer-python

I'm not sure fully context of how this should be handled either through changes to the singer-python code or if this is something that is already available in jsonschema but I am missing something. 

Any suggestions? I don't know that I have the capacity to look further into the transform code at this time unfortunately because it is outside the scope of this tap.